docs/getting-started-monitoring.md

```diff
@@ -278,6 +293,15 @@ First, you need to install the following tools used in this guide.
                  kubectl get secrets scalar-monitoring-grafana -n monitoring -o jsonpath='{.data.admin-password}' | base64 -d
                  ```
 
+1. Access log browser
+   - Go to Grafana http://localhost:3000
+   - Move to `Configuration` and choose `Data Sources`
+   - Click `Add data source`
+   - Select `Loki`
+   - Input `http://loki-stack:3100` to URL
+   - Click `Save and Add`
```

Comment:
```suggestion
   - Click `Save & test`
```
In my local test environment (using Grafana included in the latest kube-pormetheus-stack), the button seems `Save & test`.

In your environment, is the button `Save and Add`?
If this is the different depends on environment (version?), I think we don't need to keep consistent completely (We don't need to apply this suggestion).
Because, I think it is minor different and I think user can understand what this document state.

Reply:
Sorry, my bad.
The button is `Save and test` not `Save and Add`.
I revised in https://github.com/scalar-labs/helm-charts/pull/87/commits/64af41bb75801dfb5e74f245074b1485e5869788